### PR TITLE
Add T regression notebook

### PR DESCRIPTION
**Pull request by tjburch**
Issue #370 suggested that a t regression example notebook is needed. I recreated the example from the [pymc](https://docs.pymc.io/notebooks/GLM-robust.html) docs, and adapted it using bambi, and included it in the notebook in this commit. 

The text was largely changed and were a few minor and stylistic changes but nothing too major. I additionally added a couple of code blocks comparing the recovered parameters and evaluating LOO of the normal regression model vs the T regression.

### CODE REVIEW

File: docs/notebooks/t_regression.ipynb [hunk 1]
```diff
@@ -0,0 +1,896 @@
+{
```
[the remark this answers is not in the document]
Comment (tjburch) [reply docs/notebooks/t_regression.ipynb:1]: <p>Added:</p><blockquote>The T distribution is specified by a number of degrees of freedom (nu). In&nbsp;<code><a href="https://numpy.org/doc/stable/reference/random/generated/numpy.random.standard_t.html" target="_blank">numpy.random.standard_t</a></code>&nbsp;this is the parameter&nbsp;<code>df</code>, in the&nbsp;<a href="https://docs.pymc.io/api/distributions/continuous.html#pymc3.distributions.continuous.StudentT" target="_blank">pymc T distribution</a>, it's&nbsp;<code>nu</code>. It is constrained to real numbers greater than 0. As the degrees of freedom increase, the probability in the tails Student T distribution decrease. In the limit of&nbsp;nu-&gt;inf, the Student T distribution is a normal distribution. Below, the T distribution is plotted for various&nbsp;nu</blockquote><p>Along with a plot for nu=0.1, 1, 10 atop a normal distribution</p><p> </p>

[the remark this answers is not in the document]
Comment (tjburch) [reply docs/notebooks/t_regression.ipynb:1]: <p>Added "mean"</p>

[the remark this answers is not in the document]
Comment (tjburch) [reply docs/notebooks/t_regression.ipynb:1]: <p>Added</p><blockquote>Additionally, this inference is based on sampling methods, so it is expected to have small differences (especially given a relatively small number of samples).</blockquote>

[the remark this answers is not in the document]
Comment (tomicapretto) [reply docs/notebooks/t_regression.ipynb:1]: <p>Excellent job!</p>

[the remark this answers is not in the document]
Comment (tomicapretto) [reply docs/notebooks/t_regression.ipynb:1]: <p>Great!</p>

File: docs/notebooks/t_regression.ipynb [hunk 2]
```diff
@@ -0,0 +1,946 @@
...
```
[the remark this answers is not in the document]
Comment (tjburch) [reply docs/notebooks/t_regression.ipynb:1]: Thanks for tanking a look. Done in latest commit.

**Pull request merged**